util/opentelemetry-util-genai/src/opentelemetry/util/genai/span_utils.py

```diff
@@ -112,6 +93,8 @@ def _apply_finish_attributes(span: Span, invocation: LLMInvocation) -> None:
     _maybe_set_span_messages(
         span, invocation.input_messages, invocation.output_messages
     )
+    _apply_request_attributes(span, invocation)
```

Comment:
not blocking for this PR, but we should eventually allow to set attributes at start time. I noticed we don't properly mark which attributes should be provided at the start time, fixing it in the https://github.com/open-telemetry/semantic-conventions/pull/2994